docs/pages/eas/workflows/syntax.mdx

```diff
@@ -384,6 +384,33 @@ jobs:
       prod: boolean # optional
 ```
 
+#### `fingerprint`
+
+```yaml
+jobs:
+  my_job:
+    # @info #
+    type: fingerprint
+    # @end #
+```
+
+Calculates fingerprint of iOS and Android builds.
```

Comment:
```suggestion
Calculates fingerprint of iOS and Android builds.
```

Can you move this line at the top of this section? Currently, the structure we follow is:
- Definition
- Syntax
- Context (optional)
- Example (optional)